backend/substrapp/utils/errors.py

```diff
@@ -2,27 +2,29 @@
 
 from django.core import files
 
-from builder import exceptions as builder_errors
 from substrapp import models
 from substrapp import utils
+
 from substrapp.compute_tasks import errors as compute_task_errors
 
 
-def store_failure(exc: Exception, compute_task_key: str) -> Optional[models.ComputeTaskFailureReport]:
+def store_failure(
+    exception, asset_key: str, asset_type: models.FailedAssetKind, error_type: compute_task_errors.ComputeTaskErrorType
+) -> Optional[models.AssetFailureReport]:
     """If the provided exception is a `BuildError` or an `ExecutionError`, store its logs in the Django storage and
     in the database. Otherwise, do nothing.
 
     Returns:
-        An instance of `models.ComputeTaskFailureReport` storing the error logs or None if the provided exception is
+        An instance of `models.AssetFailureReport` storing the error logs or None if the provided exception is
         neither a `BuildError` nor an `ExecutionError`.
     """
 
-    if not isinstance(exc, (compute_task_errors.ExecutionError, builder_errors.BuildError)):
+    if error_type == compute_task_errors.ComputeTaskErrorType.INTERNAL_ERROR:
```

Comment:
I'm wondering if it's not that kind of internal errors that leads to the "please contact your administrator for full logs" message on the frontend. 
If we want to change that behaviour, we might have to log all errors. 
What do you think?

Reply:
Interestign topic, but I think, as small as the changes are, go further than the scope of this PR

Reply:
Agreed, it's definitely something we want to do in its own PR 👍